api/control/v1alpha1/types.go

```diff
@@ -153,7 +153,19 @@ type DownsyncPolicyClause struct {
 	// +optional
 	CreateOnly bool `json:"createOnly,omitempty"`
 
-	// statusCollectors is a list of StatusCollectors name references that are applied to the selected objects.
+	// `statusCollection` holds the rules for collecting status from the WECs.
```

Comment:
Why introduce this level of structure for BindingPolicy objects and not for Binding objects?